packages/backend-deployer/src/cdk_deployer.ts

```diff
@@ -161,23 +164,52 @@ export class CDKDeployer implements BackendDeployer {
     // actionable errors being hidden among the stdout. Moreover execa errors are
     // useless when calling CLIs unless you made execa calling error.
     let aggregatedStderr = '';
-    const aggregatorStream = new stream.Writable();
-    aggregatorStream._write = function (chunk, encoding, done) {
+    const aggregatorStderrStream = new stream.Writable();
+    aggregatorStderrStream._write = function (chunk, encoding, done) {
```

Comment:
I didn't realize that aggregating output is existing thing.
Mind logging GH issue to make this part "streaming and parsing errors" ?

Reply:
Will log it, this would be a little harder since we are actually throwing and processing elsewhere.

Reply:
https://github.com/aws-amplify/amplify-backend/issues/662